integrationtests/org.eclipse.xtext.swtbot.testing/src/org/eclipse/xtext/swtbot/testing/api/EclipseAPI.java

```diff
@@ -35,6 +37,12 @@ private EclipseAPI() {
 	}
 
 	public static MainMenuAPI mainMenu() {
+		UIThreadRunnable.syncExec(new VoidResult() {

```

Comment:
why is this needed? is there a better place?

Reply:
Currently executing the tests again and noticed that they do not run in full-screen-mode ... maybe the code in AbstractSwtBotTest is not executed for som ereason ...

Reply:
Guess instead of this we have to fix the usage of junit here ... remove the @BeforeClass in the abstract superclass and explicitely call it from subclass ...

Reply:
Quoting the JavaDoc of BeforeClass:

> The <code>@BeforeClass</code> methods of superclasses
>  will be run before those of the current class, unless they are shadowed in the current class.
 
 Not sure if that's the real issue.